src/libraries/Microsoft.PowerFx.Core/Public/Values/DateTimeValue.cs

```diff
@@ -50,21 +50,21 @@ internal static DateTime GetConvertedDateTimeValue(DateTime value, TimeZoneInfo
             }
             
             // Since we can't convert LocalKind time to UTC, if the time was of kind local just change kind.
-            if (value.Kind == DateTimeKind.Local && timeZoneInfo.BaseUtcOffset == TimeSpan.Zero)
```

Comment:
>BaseUtcOffset == TimeSpan.Zero) [](http://example.com/codeflow?start=65&length=31)

Just curious - 
 Why was original code wrong?

Reply:
nit:

Maybe a IsUtc() extension method? 
(that also gives a place to comment why not to use != TimeSpan.Zero

Reply:
Today, in London, BaseUtcOffset is zero. But not throughout the entire year.